tests/unit/core/components/TestServerHtml.js

```diff
@@ -85,7 +93,7 @@ describe(__filename, () => {
     const root = render();
     const js = root.find('script');
 
-    expect(js.at(1)).toHaveProp('src', '/foo/disco-blah.js');
+    expect(js.at(2)).toHaveProp('src', '/foo/disco-blah.js');
```

Comment:
It's not clear to me why this has changed..?

Reply:
There is now a script injected before those scripts. It is the script tag for the JSON-LD data.